notebooks/en/pipeline_notus_instructions_preferences_legal.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Maybe explain here that it could be possible to do feedback with a non OpenAI model but performance might be lower? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>